components/sidebar/sidebar_channel/channel_mention_badge/index.ts

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
There's a blank line here.


Also, are you expecting that we'll connect this component? I don't want to leave a bunch of extra code around that we don't plan on using.

Reply:
Just leaving it in case we need to. I'll remove it when i'm sure we don't need to connect it.